doc/source/user_guide/scale.rst

```diff
@@ -374,5 +374,101 @@ datasets.
 
 You see more dask examples at https://examples.dask.org.
 
+Use Modin
+---------
+
+Modin_ is a scalable dataframe library, which has a drop-in replacement API for pandas and
+provides the ability to scale pandas workflows across nodes and CPUs available and
+to work with larger than memory datasets. To start working with Modin you just need
```

Comment:
nit: This is a run on sentence. Maybe just split off the last part?

> Modin_ is a scalable dataframe library, which has a drop-in replacement API for pandas and provides the ability to scale pandas workflows across nodes and CPUs available. It is also able to work with larger than memory datasets.

Reply:
Done, thanks for the suggestion.